src/templates/next/components/shared/ArticlePageHeader/ArticlePageHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import type { ArticlePageHeaderProps } from "~/common"
+import Breadcrumb from "../../Breadcrumb"
+import BaseParagraph from "../Paragraph"
+import { Paragraph } from "../../../typography/Paragraph"
+
+const ArticlePageHeader = ({
+  breadcrumb,
+  category,
+  title,
+  date,
+  summary,
+  LinkComponent,
+}: ArticlePageHeaderProps) => {
+  return (
+    <div className="px-4 lg:px-5">
+      <div className="max-w-[960px] mx-auto">
+        <div className="my-8 lg:my-16">
+          <Breadcrumb links={breadcrumb.links} LinkComponent={LinkComponent} />
+        </div>
+
+        <div className="mb-3 text-lg text-content-medium">{category}</div>
+
+        <div className="flex flex-col gap-9 max-w-[660px]">
```

Comment:
whats our stance here ah? should we not be defining these `max-w-[660px]` in presets?

Reply:
For the layouts-level stuff I don't think we have fully finalised the list of breakpoints/container widths yet, so this is hard-coded for now pending confirmation from design.